scene/main/viewport.h

```diff
@@ -234,6 +234,7 @@ class Viewport : public Node {
 	Size2i size;
 	Size2i size_2d_override;
 	bool size_allocated = false;
+	bool xr = false;
```

Comment:
Maybe the property could be renamed `use_xr`?

Reply:
yeah I'm not sure to be honest... xr is a little non-descriptive indeed...

Reply:
If I might suggest something: `use_arvr`? I understand you're using `xr` here as a umbrella term for everything like that, but if you're concerned about the potential for it to be misunderstood, then using `arvr` may be a better solution.

Though, personally, I don't actually think `use_xr` would be a bad choice, even if it seems non-descriptive.

Reply:
We actually went from it being arvr to it being xr because people found arvr too confusing and xr seems to be the accepted moniker now :)

Reply:
I'm stuck between `enable_xr` and `use_xr`

Reply:
I settled on 'use_xr'